pkg/tracking/jira/client_test.go

```diff
@@ -154,7 +154,7 @@ func TestClient_Get(t *testing.T) {
 			setupSubTestClient(t)
 
 			got, err := jiraClient.GetTicket(tt.ticketID)
-			if errToStr(err) != errToStr(tt.wantErr) {
+			if !isSameError(err, tt.wantErr) {
```

Comment:
I think this can be simplified to:
```go
if !strings.Contains(testutil.ErrToStr(err),testutil.ErrToStr(tt.wantErr)) 

``` 
and, as bonus point, we could also get rid of the function ``isSameError``.
WDYT?

Reply:
I had a lot of doubts about how to handle this kind of comparisons between errors and wrapped errors. This solutions is ok for me. But maybe, this type of checks are too fragile and we should think in another approach for error message checks.